public_service_finder/public_service_finder/settings.py

```diff
@@ -0,0 +1,125 @@
+"""
+Django settings for public_service_finder project.
+
+Generated by 'django-admin startproject' using Django 5.1.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-qkdoa^_d-cnf1)7u!pvlq1i6s9^5(__iaiczz94&e4hz1@4=26'
```

Comment:
Lets discuss this (Not a blocker though)

Reply:
Made the changes! Thanks!